pkg/datastore/filedb/filedb.go

```diff
@@ -84,3 +130,18 @@ func (f *FileDB) Update(ctx context.Context, col datastore.Collection, id string
 func (f *FileDB) Close() error {
 	return f.backend.Close()
 }
+
+func makeHotStorageFilePath(kind, id string, shard datastore.Shard) string {
+	return fmt.Sprintf("%s/%s/%s.json", kind, shard, id)
```

Comment:
I think we have to spit data by project ID to avoid fetching data from all projects.
So project ID must be specified for every command.

Reply:
Nice catch 👍  Tbh, I thought about it too, but in such case, we have to know the projectID in all cases, that changes the current implementation on datastore layer a lot. Can I address it by another PR later, since it's not really related to the purpose of this PR, but ofcourse we have to handle it as you mentioned 🙏

Reply:
Sure. So please add a TODO for that issue.